server/flyway/sql/V51__add_user_terms_conditions_table.sql

```diff
@@ -0,0 +1,47 @@
+-- Create fam_user_terms_conditions table
+CREATE TABLE IF NOT EXISTS app_fam.fam_user_terms_conditions
+(
+    user_terms_conditions_id        bigint          GENERATED BY DEFAULT AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    user_id                         bigint          NOT NULL,
+    version_id                      int             NOT NULL,
```

Comment:
I am thinking maybe column name just give it as "**version**".
With "**id**" it makes me confused that it is a system generated `id` or it is a foreign key to a real "**terms_and_conditions**" table's id, but we are not modeling that currently to store terms and condition text in backend.

However, I am wondering if the "**version**" column should be an "int" or "varchar(30)" type as I am wondering the format of version is simply just "1, 2, 3" or "1.0.1" or "v1.0.1", "FAM:DAMIN:V2024-05-19"or  something else. We didn't discuss that...
Perhaps @basilv knows the T&C does have version number(or string) format?

Reply:
yeah I agree Ian, I was thinking to keep it simple for now, but that's a good point, we can ask tomorrow. I'll change it to "version" first, or "terms_conditions_version"?

Reply:
I'd say use string (varchar) for version as data type. Agreed this should be called version instead of version_id.
We will just use version = "1" initially.